pkg/ncmpcpp

```diff
@@ -0,0 +1,38 @@
+[mirrors]
+http://github.com/arybczak/ncmpcpp/archive/b7da5d1b9af1f3b0022c1f7daa0c7d12ae72a21d.tar.gz
+
+[vars]
+filesize=219516
+sha512=8dd78f463811826729cb1498a7ccc4bff49393e88adb80260ad499623cf4ecdce5b405c1d91ca03f0b34bb7f5329124cb7a0541dbb880be2a634b057af93b9aa
+pkgver=1
+tardir=ncmpcpp-b7da5d1b9af1f3b0022c1f7daa0c7d12ae72a21d
+
+[deps]
+boost
+gnu-readline
+automake
+
+[deps.run]
+libmpdclient
```

Comment:
does libmpdclient ship executables that are called at runtime, unlike its name suggests ? if so, this is the right section to put it. but if it is a library, which code is linked against at build time, it should be in [deps]